### PR TITLE
PLAT-1312 Remove Support for Legacy Service URLs

### PR DESCRIPTION
**Pull request by oliver-richers**

### CODE REVIEW

File: platform-common/src/main/java/com/softicar/platform/common/network/http/HttpStatusCode.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	}
+
+	public int getCode() {
+
+		return code;
+	}
+
+	public boolean isInformational() {
+
+		return code > 100 && code < 200;
```
Comment (alexander-schmidt) [inline platform-common/src/main/java/com/softicar/platform/common/network/http/HttpStatusCode.java:66]:
```suggestion
		return code >= 100 && code < 200;
```

File: platform-core-module/src/main/java/com/softicar/platform/core/module/web/service/WebServiceBrokerServiceExceptionHandler.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package com.softicar.platform.core.module.web.service;
+
+import com.softicar.platform.common.core.utils.CastUtils;
+import com.softicar.platform.common.core.utils.DevNull;
+import com.softicar.platform.common.network.http.HttpStatusCode;
+import com.softicar.platform.common.network.http.error.HttpError;
+import java.io.IOException;
+import javax.servlet.http.HttpServletResponse;
+
+public class WebServiceBrokerServiceExceptionHandler {
```
Comment (alexander-schmidt) [inline platform-core-module/src/main/java/com/softicar/platform/core/module/web/service/WebServiceBrokerServiceExceptionHandler.java:10]: Neither this class nor `WebServiceBrokerService` (in which this class is used) is covered by a unit test. We should fix that.

**Review by alexander-schmidt — CHANGES_REQUESTED** [review]

File: platform-common/src/main/java/com/softicar/platform/common/network/http/HttpStatusCode.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
+package com.softicar.platform.common.network.http;
+
+/**
+ * Enumerates all HTTP (error) codes.
```
[the remark this answers is not in the document]
Comment (oliver-richers) [reply platform-common/src/main/java/com/softicar/platform/common/network/http/HttpStatusCode.java:4]: yes, sure

File: platform-common/src/main/java/com/softicar/platform/common/network/http/HttpStatusCode.java (hunk 1 above)
Comment (oliver-richers) [reply platform-common/src/main/java/com/softicar/platform/common/network/http/HttpStatusCode.java:66]: oups

File: platform-core-module/src/main/java/com/softicar/platform/core/module/web/service/WebServiceBrokerServiceExceptionHandler.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+		exception.printStackTrace();
+		sendError(HttpStatusCode.INTERNAL_SERVER_ERROR, "Internal Server Error");
+	}
+
+	private void sendError(HttpStatusCode statusCode, String message) {
+
+		try {
+			response.sendError(statusCode.getCode(), message);
+		} catch (IOException exception) {
+			// ignoring I/O exceptions; probably the client closed the connection
```
[the remark this answers is not in the document]
Comment (oliver-richers) [reply platform-core-module/src/main/java/com/softicar/platform/core/module/web/service/WebServiceBrokerServiceExceptionHandler.java:44]: ok

**Review by alexander-schmidt — APPROVED** [review]

**Pull request merged**